api/v1/swagger.yaml

```diff
@@ -4000,6 +4016,13 @@ paths:
                 description: >
                   Max 60 characters; case insensitive; allows alpha-numeric characters,
                   underscore (_) and dash (-).
+              sortBy:
```

Comment:
sure. sounds good! Will include the description change as part of story 3 - view layer. Thank you